### ISSUE TITLE
2022-05-18T09:57:47.836Z [INFO]: Must use import to load ES Module: /root/.nvm/versions/node/v14.19.0/lib/node_modules/@aws-amplify/cli/node_modules/libphonenumber-js/index.common.js                                  require() of ES modules is not supported.

### ISSUE THREAD

**Issue by jjrdevspace01** [aws-amplify/amplify-cli#10437]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

### How did you install the Amplify CLI?

amplify.yml

### If applicable, what version of Node.js are you using?

v14.19.0

### Amplify CLI Version

8

### What operating system are you using?

amplify console

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

no

### Amplify Categories

custom

### Amplify Commands

push

### Describe the bug

amplify.yml contains
```
backend:
  phases:
    build:
      commands:
        - amplifyPush --simple
```
backend deploys fail with
Must use import to load ES Module: /root/.nvm/versions/node/v14.19.0/lib/node_modules/@aws-amplify/cli/node_modules/libphonenumber-js/index.common.js
                                 require() of ES modules is not supported.

as a workaround
```
```
backend:
  phases:
    build:
      commands:
        - npm install -g @aws-amplify/cli@7
        - amplifyPush --simple
```
```

### Expected behavior

backend deploys as normal

### Reproduction steps

1. produce amplify.yml as above
2

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

**Comment by parvusville** [comment]
Facing the same problem, using CLI 7.5.4. `amplfiy push` from the CLI seems to work.

**Comment by fossamagna** [comment]
[libphonenumber-js@1.10.3](https://www.npmjs.com/package/libphonenumber-js/v/1.10.3) has been released. It may have an impact that added ESM support in v1.10.0.

See: https://gitlab.com/catamphetamine/libphonenumber-js/-/blob/master/CHANGELOG.md#1100-18052022

**Comment by raffclar** [comment]
Same issue with @aws-amplify/cli@7.6.9

**Comment by wvidana** [comment]
Yes @fossamagna , it looks like it. The problem is that only the most recent CLI locks down `libphonenumber-js` to a previous version
https://github.com/aws-amplify/amplify-cli/blob/1b789bfa5e84ac3cf9c7e9b55e456bb01f07aafc/packages/amplify-appsync-simulator/package.json#L44

Older Amplify CLIs just let a bigger version to be installed
https://github.com/aws-amplify/amplify-cli/blob/aa1096ca504bdb7e6a2dca2963c546f957116f9d/packages/amplify-appsync-simulator/package.json#L44

I'm still trying to figure out how to lock that version, since when I try to install the package globally with `npm i -g libphonenumber-js@1.9.47` that doesn't affect the amplify-cli install of the dependency:
```
├─┬ @aws-amplify/cli@6.4.0
│ ├─┬ amplify-provider-awscloudformation@4.65.0
│ │ └─┬ @aws-amplify/graphql-default-value-transformer@0.3.0
│ │   └── libphonenumber-js@1.10.3
│ └─┬ amplify-util-mock@3.34.12
│   └─┬ amplify-appsync-simulator@1.27.8
│     └── libphonenumber-js@1.10.3 deduped
└── libphonenumber-js@1.9.52
```
As you can see, the `libphonenumber-js` installed alongside Amplify is >1.10, even when I installed  before installing the 6.4.0 version... Maybe it's because it was already shipped in the Dockerfile?

Anyone has any idea on how to override the shipped version?

**Comment by manny-kunaico** [comment]
Looks like this is fixed in the `libphonenumber-js` library! https://gitlab.com/catamphetamine/libphonenumber-js/-/issues/62

I was about to implement the suggestion made by @wvidana and wanted to do a negative test first before implementing the fix, but the negative test passed since the library was updated to fix the issue 😄

**Comment by ykethan** [comment]
Hey, Thank you for raising this. Based on the gitlab issue this appears to have been fixed. Please do let us know if you are still experiencing this behaviour.
